src/drawing/__tests__/DrawingTarget-test.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { MutableRefObject } from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import DrawingTarget from '../DrawingTarget';
+import { annotations } from '../__mocks__/data';
+import { getShape } from '../drawingUtil';
+
+const { target: mockTarget } = annotations[0];
+
+jest.mock('react', () => ({
+    ...jest.requireActual('react'),
+    useState: jest.fn(),
```

Comment:
It looks like the component no longer uses state. Is this still needed? If so, we should be able to accomplish it via a spy in `beforeEach`, instead.